src/k8s/cmd/k8s/k8s_bootstrap.go

```diff
@@ -13,11 +13,13 @@ import (
 	"github.com/canonical/k8s/cmd/k8s/errors"
 	"github.com/canonical/k8s/pkg/utils"
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
 )
 
 var (
 	bootstrapCmdOpts struct {
 		interactive bool
+		config      string // yaml config filename
```

Comment:
configFileName instead?

Reply:
I'm not sure here if this has to be the same as the flag name. Going to see now when I build the snap...